rfc8724.xml

```diff
@@ -215,8 +215,8 @@ The L2 Word size is assumed to be constant over time for each device.</dd>
 <!--[rfced] Is there a reason "Padding (P)" does not appear as "P:
 Padding" in the Terminology section to match the other entries?
 May we update for consistency?  -->
-	
-        <dt>Padding (P):</dt><dd>Extra bits that may be appended by SCHC to a data unit that it passes to the underlying Layer 2 for transmission.
+<!--[db] the abbreviation P is actually not used anywhere in the document -->	
+        <dt>Padding:</dt><dd>Extra bits that may be appended by SCHC to a data unit that it passes to the underlying Layer 2 for transmission.
```

Comment:
the abbreviation P is actually not used anywhere in the document

Reply:
And as I told you in my mail you  have: pad, Pad, padding
I propose to use pad

Reply:
Thanks, Ana!
"Pad" is gone (there was only one, it was a mistake).
"pad" is only left in two or three drawing because of space limitation, as an abbreviation of "padding". I dont think it justifies to appear as such in the terminology.